modules/imaging_uploader/php/NDB_Menu_Filter_imaging_uploader.class.inc

```diff
@@ -230,8 +205,8 @@ class NDB_Menu_Filter_Imaging_Uploader extends NDB_Menu_Filter_Form
         $file   = new File_Upload;
         $config = NDB_Config::singleton();
         $paths  = $config->getSetting('paths');
-        $file->overwriteMode = "reject";
-        $file->fileMove      = false;
+        $file->setOverwriteMode("overwrite");
```

Comment:
@alisterdev 

Maybe in a separate pull request, could you please set this mode to "reject" (just so that the overwrite behavior does not change from what we currently have)? 
or if you overwrite (as in the details of Redmine 11639), the user needs to be informed of that with a popup message I guess, and the mri_upload table entry updated. Currently it overwrites "silently" and creates a new entry in the mri_upload table.